lib/puma/plugin/statsd.rb

```diff
@@ -117,11 +117,13 @@ def register_hooks
     in_background(&method(:stats_loop))
   end
 
-  def fetch_stats
-    stats = Puma.stats
-    if stats.is_a?(Hash)
-      stats
-    else
+  if defined?(Puma.stats_hash)
```

Comment:
could this also be `if Puma.responds_to?(:stats_hash)`, and if so are you open to using that form?

Reply:
Yep! Done.